### DISCUSSION TITLE
Expected data storage usage? (clickhouse consuming ~20gb on low traffic site)

### DISCUSSION THREAD

**Discussion by Billiam** [plausible/analytics#1558] [Self-Hosted Support]
I'm using plausible on pretty low-traffic site (< 50 requests/day), and haven't done much customization nor goal setup. It's been running for a few months, and clickhouse data storage is currently consuming about 20gb of disk space (making it unusable on it's current host).

Is this typical? If not, what can be done to reduce storage usage?

**Comment by ukutaht** [comment]
Probably caused by logging in Clickhouse. Check this out https://github.com/plausible/analytics/discussions/588
